src/package.json

```diff
@@ -60,8 +60,6 @@
     "chart.js": "^2.9.3",
     "d3": "^5.9.2",
     "dragula": "^3.7.2",
-    "ng2-file-upload": "^1.3.0",
-    "ngx-mask": "^2.5.3",
```

Comment:
Because this is a breaking change, you should update BREAKING_CHANGES.md in both repositories with details on how to upgrade.